src/test/java/edu/kpi/testcourse/rest/ShortenLinkTest.java

```diff
@@ -27,7 +31,22 @@
 
   private static EmbeddedServer server;
   private static HttpClient client;
-  private static final Gson g = new Gson();
+
+  @Inject
+  JsonToolJacksonImpl jsonTool;
```

Comment:
The same as in UrlController
```suggestion
  JsonTool jsonTool;
```